src/libs/users.ts

```diff
@@ -43,10 +40,7 @@ async function getUser(db: Db, user: string) {
  * @param privateInfo Whether to include more sensitive information like calendar URLs. Defaults to false.
  * @returns An object containing user information.
  */
-export async function getInfo(db: Db, user: string, privateInfo: boolean) {
-	if (typeof db !== 'object') { throw new Error('Invalid database connection!'); }
-	if (typeof privateInfo !== 'boolean') { privateInfo = false; }
```

Comment:
Do we want to default `privateInfo` to false or no

Reply:
Totally up to you, I was on the fence with this one as well because it was also explicitly passed in everywhere it's used.
![Screenshot from 2019-12-31 13-53-50](https://user-images.githubusercontent.com/11001606/71632638-1c3cda80-2bd5-11ea-939f-117af237e9aa.png)

Reply:
Yeah we probably don't need any default parameters here